### DISCUSSION TITLE
LDAP/AD Authentication

### DISCUSSION THREAD

**Discussion by mathieumd** [sheltermanager/asm3#1352] [Ideas]
Is it possible to connect ASM3 to an LDAP/Active Directory server?

It looks like [ASM2 could authenticate to LDAP](https://sheltermanager.com/repo/asm2_help/node124.html), but could not find relevant info about ASM3.

**Comment by bobintetley** [comment]
It isn't, we never bothered implementing LDAP for asm3 I'm afraid.
